composer.json

```diff
@@ -25,7 +25,7 @@
         "php":                      "^7.0",
         "phpspec/prophecy":         "^1.5",
         "phpspec/php-diff":         "^1.0.0",
-        "sebastian/exporter":       "^2.0",
+        "sebastian/exporter":       "^3.0",
```

Comment:
please allow both versions in parallel instead, to be nicer for other tools installed in projects (should be dead easy as there is no BC break except the min PHP version)